rasa/core/channels/channel.py

```diff
@@ -414,7 +416,7 @@ def stream_response(
             )
             result = None  # declare variable up front to avoid pytype error
             while True:
-                result = await q.get()
+                result = await q.get()  # pytype: disable=bad-return-type
```

Comment:
though we don't need this comment anymore after you fixed that in your previous pytype pr?